components/tidb_qe/src/codec/mysql/json/json_cast.rs

```diff
@@ -0,0 +1,63 @@
+// Copyright 2017 TiKV Project Authors. Licensed under Apache-2.0.
```

Comment:
Don't mix other refactoring with this PR.

Reply:
Looks like this is a merge issue. Orz.

Reply:
This file has been deleted.